numpy/core/src/multiarray/array_method.c

```diff
@@ -284,6 +337,9 @@ fill_arraymethod_from_slots(
             case NPY_METH_unaligned_contiguous_loop:
                 meth->unaligned_contiguous_loop = slot->pfunc;
                 continue;
+            case NPY_METH_get_identity:
+                meth->get_identity = slot->pfunc;
+                continue;
```

Comment:
Would be good to have tests for this!

Reply:
Yeah, I am using it in the unit dtype: https://github.com/seberg/unitdtype  So it is half tested, but not directly.  I guess I could add a test using the "scaled float"

Reply:
Would still be a good idea...